src/idpi/operators/support_operators.py

```diff
@@ -73,15 +73,12 @@ def init_field_with_vcoord(
     #       in the interface
     # attrs
     attrs = parent.attrs.copy()
-    attrs["GRIB_typeOfLevel"] = vcoord.type_of_level
-    if "GRIB_NV" in attrs:
```

Comment:
why do we not need this update anymore?

Reply:
The `GRIB_` attributes were set by cfgrib and no longer exist. The `NV` attribute describes the number of hybrid levels and is only relevant if we were to write some field defined on hybrid levels to grib. In any case, it can be derived from the vertical coordinates I think.

Reply:
👍